src/cljs/main/broadfcui/auth.cljs

```diff
@@ -214,19 +214,30 @@
         :not-active [:div {:style {:color (:exception-reds style/colors)}}
                      "Thank you for registering. Your account is currently inactive."
                      " You will be contacted via email when your account is activated."]
-        [:div {:style {:color (:state-exception style/colors)}}
-         "Error loading user information. Please try again later."])])
+        [:div {}
+         [:div {:style {:color (:state-exception style/colors) :paddingBottom "1rem"}}
+          "Error loading user information. Please try again later."]
+         [:table {:style {:color (:text-lighter style/colors)}}
+          [:tbody {:style {}}
+           [:tr {} [:td {:style {:fontStyle "italic" :textAlign "right" :paddingRight "0.3rem"}} "What went wrong:"] [:td {} (:message (:error @state))]]
+           [:tr {} [:td {:style {:fontStyle "italic" :textAlign "right" :paddingRight "0.3rem"}} "Status code:"] [:td {} (:statusCode (:error @state))]]]]])])
```

Comment:
consider using the `ErrorViewer` component here, to enable easy expanding/hiding of stack traces.

Reply:
The `ErrorViewer` doesn't have an easy way to auto expand (perhaps by environment), and I felt it was fairly important for status codes to be visible in screenshots (they are in the expandable section)

Will do some more tinkering and see if we can do better